docs/content/what-is-a-local-activity.md

```diff
@@ -6,13 +6,19 @@ description: todo
 
 A Local Activity is an [Activity Execution](/docs/content/what-is-an-activity-execution) that executes in the same process as the [Workflow Execution](/docs/content/what-is-a-workflow-execution) that spawns it.
 
-Although a Local Activity consumes less resources than a regular Activity Execution, it is subject to shorter durations and a lack of rate limiting.
+Some Activities are very short lived and do not need the queuing semantic, flow control, rate limiting and routing capabilities.
+For this case, Temporal supports a _local Activity_ feature.
```

Comment:
Here and in next sentence: Should we standardize on init-capping "Local Activity", @flossypurse?

Reply:
```suggestion
For this case, Temporal supports a _Local Activity_ feature.
```